docs/configuration/features/sequence_features.md

```diff
@@ -270,6 +290,7 @@ last vector of the sequence dimension).
 confidence_penalty: 0, robust_lambda: 0}`): is a dictionary containing a loss `type`. The only available
 loss `type` for sequences is `softmax_cross_entropy`. For more details on losses and their options, see also
 [Category Output Features and Decoders](../category_features#category-output-features-and-decoders).
+- **`decoder`** (default: `{"type": "generator"}`): Decoder for the desired task. Options: `generator`, `tagger`.
```

Comment:
Worth adding a description for what `generator` and `tagger` are, and when to use them?

Reply:
Added links.